### PR TITLE
[BE-2793][BE-2537] Clean up examples: remove AWS credentials, distributed flags, etc.

### PR DESCRIPTION
**Pull request by ehsantn**

### CODE REVIEW

File: ml/credit-card-fraud.ipynb [hunk 1]
```diff
@@ -979,10 +1002,17 @@
...
+     "output_type": "stream",
+     "text": [
+      "%px:   0%|          | 0/8 [00:00<?, ?tasks/s]"
+     ]
+    },
     {
      "data": {
       "text/plain": [
-       "[stdout:5] [17:39:17] WARNING: /Users/runner/work/xgboost/xgboost/src/learner.cc:1115: Starting in XGBoost 1.3.0, the default evaluation metric used with the objective 'binary:logistic' was changed from 'error' to 'logloss'. Explicitly set eval_metric if you'd like to restore the old behavior.\n"
+       "[stdout:2] [18:18:11] WARNING: /Users/ehsan/dev/tmp/xgboost/python-package/build/temp.macosx-10.9-x86_64-cpython-39/xgboost/src/learner.cc:1115: Starting in XGBoost 1.3.0, the default evaluation metric used with the objective 'binary:logistic' was changed from 'error' to 'logloss'. Explicitly set eval_metric if you'd like to restore the old behavior.\n"
```
Comment (hadia206) [inline ml/credit-card-fraud.ipynb:1015]: should we remove your path and make it generic?

File: notebooks/Chicago-crimes.ipynb [hunk 2]
```diff
@@ -217,23 +139,16 @@
...
+     "output_type": "stream",
+     "text": [
+      "%px:   0%|          | 0/8 [00:41<?, ?tasks/s]"
+     ]
     },
     {
      "data": {
       "text/plain": [
-       "[stdout:0] Data cleanup time:  2002.4346562838673  (ms)\n",
+       "[stdout:0] Data cleanup time:  22520.818512000005  (ms)\n",
```
Comment (hadia206) [inline notebooks/Chicago-crimes.ipynb:151]: cleanup time went 10x the previous run. do we know why?

File: notebooks/Chicago-crimes.ipynb [hunk 3]
```diff
@@ -293,10 +215,17 @@
...
+     "output_type": "stream",
+     "text": [
+      "%px:   0%|          | 0/8 [00:06<?, ?tasks/s]"
+     ]
+    },
     {
      "data": {
       "text/plain": [
-       "[stdout:0] Getting top crimes Time:  114.37122076108608  (ms)\n",
+       "[stdout:0] Getting top crimes Time:  284.19172400003845  (ms)\n",
```
Comment (hadia206) [inline notebooks/Chicago-crimes.ipynb:228]: Same 2x slower

File: notebooks/Chicago-crimes.ipynb [hunk 4]
```diff
@@ -350,10 +272,17 @@
...
+     "output_type": "stream",
+     "text": [
+      "%px:   0%|          | 0/8 [00:06<?, ?tasks/s]"
+     ]
+    },
     {
      "data": {
       "text/plain": [
-       "[stdout:0] Filtering crimes Time:  79.46234036899114  (ms)\n",
+       "[stdout:0] Filtering crimes Time:  439.49683599998934  (ms)\n",
```
Comment (hadia206) [inline notebooks/Chicago-crimes.ipynb:285]: same? Is the data the same?

File: notebooks/parking/nyc-parking-tickets.ipynb [hunk 5]
```diff
@@ -141,23 +89,16 @@
...
+     "output_type": "stream",
+     "text": [
+      "%px:   0%|          | 0/8 [07:25<?, ?tasks/s]"
+     ]
     },
     {
      "data": {
       "text/plain": [
-       "[stdout:0] Reading Time:  11.105094708482284\n"
+       "[stdout:0] Reading Time:  395.83860127499975\n"
```
Comment (hadia206) [inline notebooks/parking/nyc-parking-tickets.ipynb:101]: time ?

Comment (hadia206) [reply notebooks/parking/nyc-parking-tickets.ipynb:101]: Same comment time and results are different than expected.

**Review by hadia206 — COMMENTED** [review]
looks good regarding removing AWS credentials. 
However, some notebooks output is different than previous runs. We need to investigate that and ensure that the numbers match Pandas.

File: ml/credit-card-fraud.ipynb (hunk 1 above)
Comment (ehsantn) [reply ml/credit-card-fraud.ipynb:1015]: I didn't pay too much attention to outputs, just focusing on fixing the code and text. I'll try to revert all output changes for now.

File: notebooks/Chicago-crimes.ipynb [hunk 6]
```diff
@@ -12,36 +12,37 @@
   },
   {
    "cell_type": "markdown",
+   "id": "ea1f302f-c922-4942-874e-b0a6787022f2",
```
[the remark this answers is not in the document]
Comment (ehsantn) [reply notebooks/Chicago-crimes.ipynb:15]: I'll revert the output for now.

File: ml/credit-card-fraud.ipynb (hunk 1 above)
Comment (ehsantn) [reply ml/credit-card-fraud.ipynb]: Made the paths generic.

File: ml/flight_cancelation_prediction.ipynb [hunk 7]
```diff
@@ -718,22 +707,72 @@
...
+       "  warnings.warn(\n"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[stderr:2] /Users/ehsan/opt/anaconda3/envs/T7/lib/python3.9/site-packages/sklearn/base.py:443: UserWarning: X has feature names, but RandomForestClassifier was fitted without feature names\n",
```
[the remark this answers is not in the document]
Comment (ehsantn) [reply ml/flight_cancelation_prediction.ipynb]: Replaced the paths and removed the warnings for now.

File: notebooks/Chicago-crimes.ipynb (hunk 3 above)
Comment (ehsantn) [reply notebooks/Chicago-crimes.ipynb]: Same.

File: notebooks/Chicago-crimes.ipynb (hunk 4 above)
Comment (ehsantn) [reply notebooks/Chicago-crimes.ipynb]: Haven't changed any input data.

File: notebooks/tpch.ipynb [hunk 8]
```diff
@@ -23,14 +23,44 @@
   },
   {
    "cell_type": "markdown",
+   "id": "275bd502-c7df-4693-ad91-32abacc66be8",
```
[the remark this answers is not in the document]
Comment (ehsantn) [reply notebooks/tpch.ipynb:26]: Same.

**Comment by ehsantn** [comment]
@hadia206 reverted the outputs for now and opened a follow up issue to investigate potential performance regressions: https://bodo.atlassian.net/browse/BE-2808

**Review by hadia206 — APPROVED** [review]
looks good to me.
Thanks a lot @ehsantn

**Pull request merged**